client/src/features/users/Auth/SetNewPassword.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import Button from '@material-ui/core/Button';
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
+
+import type { Theme } from '@material-ui/core/styles';
+
+import PasswordInput from './PasswordInput';
+import { UserContext } from '../../../providers';
+
+const useStyles = makeStyles((theme: Theme) => {
+  const xPadding = 12;
+  const yPadding = 6;
+  const yMargin = 8;
+
+  return {
+    paper: {
+      maxWidth: 821 - theme.spacing(xPadding),
+      maxHeight: 732 - theme.spacing(yPadding),
+      borderRadius: 10,
+      marginTop: theme.spacing(yMargin),
+      marginBottom: theme.spacing(yMargin),
+      paddingTop: theme.spacing(yPadding),
+      paddingBottom: theme.spacing(yPadding),
+      paddingLeft: theme.spacing(xPadding),
+      paddingRight: theme.spacing(xPadding),
+      margin: 'auto',
+    },
+    header: { fontWeight: 'bold', marginBottom: 68 },
+    button: {
+      borderRadius: 0,
+      height: 62,
+      textTransform: 'none',
+    },
+  };
+});
+
+function SetNewPassword() {
+  const classes = useStyles();
+
+  const [password, setPassword] = React.useState<string>('');
+  const [error, setError] = React.useState<string | null>(null);
+
+  const handleChange = (evt: React.ChangeEvent<HTMLInputElement>): void => {
+    setPassword(evt.target.value);
+  };
+
+  const [user] = React.useContext(UserContext);
+
+  const handleSubmit = async (evt: React.FormEvent): Promise<void> => {
+    evt.preventDefault();
+    // ternary in url temporary. User should be logged in(?) id should be available once BE is built
+    fetch(`http://localhost:3001/api/users/${user ? user.id : null}`, {
```

Comment:
Add the property to include the credentials with the request. See this as a reference: 
https://github.com/Nonprofit-Exchange-Hub/web-app/blob/b7ba0f60b77a889adbbf21755ea5fcd59b1acb0e/client/src/providers/UserProvider.tsx#L22-L27

Reply:
Thanks! Made the change. I will definitely be looking into sending a fetch with credentials and how that works.